benchmark/src/jmh/java/com/knuddels/jtokkit/DataDownloader.java

```diff
@@ -150,7 +150,7 @@ public static void main(String[] args) throws Exception {
                 "'s", "'t", "'re", "'ve", "'m", "'ll", "'d", "'x",
                 "x",
                 "123",
-                "ő",
+                "a",
```

Comment:
testing a slightly different setup here since `aaaaaa` is still a single token, so this makes it a more difficult case